app/cypress/integration/community.spec.js

```diff
@@ -140,3 +151,52 @@ describe("Community page (small screen)", () => {
     cy.findByTestId("hmw-community-map").should("not.be.visible");
   });
 });
+
+describe("Community page Show Additional Text", () => {
+  beforeEach(() => {
+    cy.visit("/community/San%20Antonio,%20TX/overview");
+  });
+
+  it("Clicking Show Text switch toggles intro text", () => {
+    cy.findByText("Your Waters: What We Know");
+    cy.findByLabelText("Show Text").click({ force: true });
+    cy.findByText("Your Waters: What We Know").should("not.exist");
```

Comment:
I know we just talked about this on the call not long ago, but since you're asserting that the "Your Waters: What We Know" element doesn't exist, this would be the perfect use case for storing the results of that selector (two lines up) via `.as()` and using `cy.get()` with the variable name here, in stead of re-querying for it in the DOM.

Reply:
When updating these lines to aliases, Cypress would try to revisit /community/San%20Antonio,%20TX/overview before checking to make sure the text would not exist. I wasn't able to figure out why this was happening.

Reply:
I'm not sure what you're seeing with Cypress revisiting that route (that only happens before every test in this `describe()` test block runs as we have it in the `beforeEach()` block)... but I was wrong that you could use a Cypress alias here – I forgot we were conditionally rendering content based on the switch state (e.g. only rendering the content if the switch is checked, so elements are removed/re-added to the DOM whenever the switch is toggled).

For all tests the general rule of thumb should be:
- If the element you're testing is removed from the DOM, store the text of your selection in a normal JS variable for reuse (just makes the code easier to read/update)
- But if what you're testing is always in the DOM (it's display property is just toggled), store the selection in a Cypress alias for reuse.

So in this case, you'll want:
```js
it('Clicking Show Text switch toggles intro text', () => {
    const heading = 'Your Waters: What We Know';
    cy.findByText(heading);
    cy.findByLabelText('Show Text').click({ force: true });
    cy.findByText(heading).should('not.exist');
  });
```